src/app/services/PeerlyServices/index.ts

```diff
@@ -0,0 +1,47 @@
+import axios, {AxiosInstance, AxiosRequestConfig} from 'axios';
+import Config from 'react-native-config';
+import AsyncStore from '../asyncStorage';
+
+export const axiosInstance: AxiosInstance = axios.create({
+  baseURL: Config.PEERLY_API_BASE_URL,
+  headers: {
+    'Accept-Version': 'application/vnd.peerly.v1',
+    'Content-Type': 'application/json',
+  },
+});
+
+import {PEERLY_LOGIN_ROUTE} from '../../constant/apiRoutes';
+import {PeerlyLoginResponse} from './types';
+
+export const apiCall = async <T, D>(config: AxiosRequestConfig<T>) => {
+  let authToken: string | null;
+  if (config.url === PEERLY_LOGIN_ROUTE) {
+    authToken = await AsyncStore.getItem(AsyncStore.AUTH_TOKEN_KEY);
+    // Temporary hardcoaded the token once all API integration complete we will remove it.
+    config.headers = {
+      'Intranet-Auth':
+        'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjo5Mjl9.6zCALizV0v58UmaJIZ16i1ASULyks6uCj8ubcRrH3dU',
```

Comment:
Even if this is a temporary fix, please don't commit the token.

Reply:
removed token